### ISSUE TITLE
tfsemb_concat does not need --layer_idx

### ISSUE THREAD

**Issue by hvgazula** [hassonlab/247-pickling#76]
make changes to code accordingly

**Comment by zkokaja** [comment]
Remove argument from parser @hvgazula

**Comment by hvgazula** [comment]
I recommend removing this altogether from the tfsemb_parser.py as we are saving embeddings from all layers now. https://github.com/hassonlab/247-pickling/blob/main/Makefile#L111

**Comment by VeritasJoker** [comment]
Yes I agree. Since you are now moving the embeddings when concatenating we don't need the argument. It should just concatenate the embeddings that are newly generated.

**Comment by hvgazula** [comment]
Computation time versus storage space. I rate the former higher.
